src/main/java/io/smartcat/cassandra/diagnostics/config/ReporterConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package io.smartcat.cassandra.diagnostics.config;
+
+import io.smartcat.cassandra.diagnostics.report.LogQueryReporter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class represents the Reporter Configuration.
+ */
+public class ReporterConfiguration {
+
+    /**
+     * A fully qualified Java class name used for reporting slow queries. {@code LogQueryReporter} is the default value.
+     */
+    public String reporter = LogQueryReporter.class.getName();
```

Comment:
@nivancevic Do we want to fallback on LogQueryReporter? In this case when you have flexibility to configure reporters, I would rather see exceptions and errors that falling back to some default configuration.